.circleci/config.yml

```diff
@@ -112,6 +114,12 @@ jobs:
           ruby_version: << parameters.ruby_version >>
           bundler_version: << parameters.bundler_version >>
           project: hyrax
+      - run:
+          command: |
+            cd .internal_test_app
+            bundle install
```

Comment:
Do we need another bundle install here?  It runs at the engine level right before this.

Reply:
I don't know; I'll remove and find out. Brace yourselves, rebases are coming!

Reply:
I want to remove this, but without the step, things fail.

Reply:
See https://app.circleci.com/pipelines/github/samvera/hyrax/3828/workflows/4011ea3f-393e-484f-a0aa-fceeaa633057/jobs/25909/steps

Reply:
Thanks for trying that out.  Not in scope of this PR, but I wonder if this is another place that circleci caching could be utilized.

Reply:
Yeah, I'd love to not run bundle install more than is necessary. This looks like it shouldn't be necessary, but I leave it up to the next person.